src/openslide-vendor-aperio.c

```diff
@@ -515,6 +529,14 @@ static bool aperio_open(openslide_t *osr,
   g_auto(GStrv) props = g_strsplit(image_desc, "|", -1);
   add_properties(osr, props);
 
+  // get icc profile size, if present
+  uint32_t icc_profile_size;
+  void *icc_profile;
+  if (TIFFGetField(ct.tiff, TIFFTAG_ICCPROFILE,
```

Comment:
Might as well use the helper here too, for consistency.

Reply:
Done!